packages/flutter_tools/test/commands/create_test.dart

```diff
@@ -100,7 +100,7 @@ void main() {
     testUsingContext('plugin project', () async {
       return _createAndAnalyzeProject(
         projectDir,
-        <String>['--template=plugin'],
+        <String>['--no-pub', '--template=plugin'],
```

Comment:
Maybe add a comment somewhere that `flutter analyze` automatically runs `flutter packages get` automatically, so we don't need to also run it here.

Reply:
I just removed the `--no-pub` when we run `_createAndAnalyzeProject`. The test execution time is unaffected.